public/app/features/search/loaders.ts

```diff
@@ -1,7 +1,8 @@
 import { backendSrv } from 'app/core/services/backend_srv';
+import { NavModel } from '@grafana/data';
 
 export const loadFolderPage = (uid: string, activeChildId: string) => {
-  const navModel = {
+  const navModel: Partial<NavModel> = {
```

Comment:
This adds a bunch of `strictNullChecks` errors below, since that makes all properties of `navModal` optional and accessing `navModel.main.text`, for example, produces `Object is possibly 'undefined'.` error. 

Maybe smth like `Pick<NavModel, 'main'>` would be better since we're only interested in `main` prop here? The `navModel.main.children.find` calls below will still need to be tweaked though, since `children` is an optional prop.